service/resource/cloudformation/delete.go

```diff
@@ -3,14 +3,76 @@ package cloudformation
 import (
 	"context"
 
+	"github.com/aws/aws-sdk-go/aws"
+	awscloudformation "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/microerror"
```

Comment:
Nitpick but we order our imports into 3 blocks. Internal packages, then external and last packages from the current project.

Reply:
Thx! good to know fixed